test/pdftotext.test.js

```diff
@@ -24,7 +24,7 @@ describe('pdftotext', function(){
     it('should throw exception', function(){
       var ptext = new pdftotext(__dirname + '/pdfs/invalidfile.pdf');
       function fn() {
-        var stdout = ptext.getTextSync();
+        return ptext.getTextSync();
```

Comment:
`getTextSync()` returns `Buffer` so you need to convert it to string before regex compare. Try this:
ptext.getTextSync().toString('utf8')